src/kubernetes_cleanup_review_apps/cleanup.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import logging
+import os
+
+import kubernetes_cleanup_review_apps.context
+
+from kubernetes_cleanup_review_apps.cleanup import (
+  cleanup_review_apps
+)
+from lib.logging import setup_logging
+
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    description=('Delete old review apps')
+  )
+  parser.add_argument(
+    'artsy_env',
+    choices=['staging', 'production'],
```

Comment:
Interesting that production is supported.

Reply:
good catch. `production` no longer makes sense. i'll remove in next PR.